File: src/main/java/com/ivaneye/intellijweb2/param/OrdersParam.java
package com.ivaneye.intellijweb2.param;

import com.ivaneye.intellijweb2.model.Orders;
import javax.validation.constraints.NotNull;

/**
 * <p>
 * OrdersParam
 * </p>
 *
 * @author wangyifan
 * @since 2018-02-12
 */
public class OrdersParam extends Orders {

    @Override
    @NotNull(message = "title不能为空")
    public String getTitle() {
        return super.getTitle();
    }
}

File: src/main/kotlin/com/ivaneye/intellijweb2/param/OrdersParam.kt
package com.ivaneye.intellijweb2.param

import com.ivaneye.intellijweb2.model.Orders
import javax.validation.constraints.NotNull

/**
 * Created by wangyifan on 18-2-19.
 */
class OrdersParam : Orders() {

    override var title: String?
        @NotNull(message = "title不能为空")
        get() = super.title
        set(value: String?) {
            super.title = value
        }
}